images/win/scripts/Installers/Install-VS.ps1

```diff
@@ -4,14 +4,16 @@
 ################################################################################
 
 $toolset = Get-ToolsetContent
+Write-Host $toolset.visualStudio.signature "111"
```

Comment:
I'm curious what exactly this exact string does?

Reply:
oops. Removed